ecology.yaml

```diff
@@ -462,7 +462,14 @@ tools:
   - name: fastq_combiner
     owner: devteam
     tool_panel_section_label: 'FASTA/FASTQ'
-
+    
+  # FROGS
+  - name: frogs
```

Comment:
where is this tool developed? Does it fulfil our security standards?

Reply:
Hey ;) This tool is developed in France, notably by colleagues from INRAe. It is proposed through usegalaxy.fr and the dev repo is there https://github.com/geraldinepascal/FROGS-wrappers/ and it seems to me this was delevopped following good practices. Maybe we can ask further informations to @mariabernard or @lecorguille?

Reply:
Recently, I enhanced the security by adding a lot of `'` but maybe I missed something  👁️ 👁️

Reply:
I followed @lecorguille recommandations and I hope that everything respects the standards but if you have other suggestions I am all "ears".
The only point that we did not find a "perfect" solution is to automatically order FROGS tools in the tool pannel. @bgruening if you have any idea I am ready to follow it.
On the [readme of FROGS-wrappers](https://github.com/geraldinepascal/FROGS-wrappers#simplest-way) we suggest to modify tool_conf.xml.
Results of FROGS are also often HTML file, that need to not be sanitize ...

Let me know if you have comments.

Reply:
@mariabernard awesome. I wanted to look at the wrapper over the next days. One thing I found is the managing of the JAVA memory settings via a hidden parameter. I think this is not needed.
The ordering of tools in the panel can be controlled better with Galaxy 21.09. So there is hope :)